tests/operator/operator.go

```diff
@@ -2970,6 +2971,18 @@ spec:
 
 			const expectedSeccompProfilePath = "/proc/1/root/var/lib/kubelet/seccomp/kubevirt/kubevirt.json"
 
+			enableSeccompFeature := func() {
+				//Disable feature first to simulate addition
+				tests.DisableFeatureGate(virtconfig.KubevirtSeccompProfile)
+				tests.EnableFeatureGate(virtconfig.KubevirtSeccompProfile)
+			}
+
+			disableSeccompFeature := func() {
+				//Enable feature first to simulate removal
+				tests.EnableFeatureGate(virtconfig.KubevirtSeccompProfile)
```

Comment:
@fossedihelm Wouldn't enabling it here be redundant? since its enabled by default, between the test specs, it will be returned as part of [`AdjustKubeVirtResource()`](https://github.com/kubevirt/kubevirt/blob/609cc2a1c2ab5e94ca588ac32d3515c0416c4d7b/tests/testsuite/kubevirtresource.go#L53)

Reply:
@enp0s3 yes, currently it is, but I 'd to prefer to cover the case if in the future it will not be enabled anymore by default.
It is a non-op right now, because it does not perform any update.
If you prefer I can remove it :)